omnigibson/scenes/scene_base.py

```diff
@@ -237,6 +240,12 @@ def _load_objects_from_scene_file(self):
             obj = create_object_from_init_info(obj_info)
             # Import into the simulator
             og.sim.import_object(obj)
+            if isinstance(obj, DatasetObject) and obj.model in BOUNDING_CUBE_OBJECTS:
+                link_names = BOUNDING_CUBE_OBJECTS[obj.model]
+                for link_name in link_names:
+                    link = obj.links[link_name]
+                    for col_mesh in link.collision_meshes.values():
+                        col_mesh.set_collision_approximation("boundingCube")
```

Comment:
remove these changes - actually Cem fixed this model, so we should be okay.

Reply:
done